packages/react/src/components/TagButton/index.test.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { render, screen, fireEvent } from '@testing-library/react';
+import TagButton from './';
+import { IconType } from '../Icon';
+import axe from '../../axe';
+
+type TagButtonProps = {
+  icon: IconType;
+  label: string;
+  value: string;
+  onClick: () => void;
+};
+
+const defaultProps: TagButtonProps = {
+  icon: 'pencil',
+  label: 'Label: ',
+  value: 'value',
+  onClick: () => console.log('test')
+};
```

Comment:
Instead of retyping the types, we can use component props.

```suggestion
const defaultProps: React.ComponentProps<typeof TagButton> = {
  icon: 'pencil',
  label: 'Label: ',
  value: 'value',
  onClick: () => console.log('test')
};
```